packages/utils/src/fhir-converter/e2e-test.ts

```diff
@@ -55,9 +54,9 @@ dayjs.extend(duration);
  * - fhirBaseUrl: the URL of the FHIR server;
  */
 
-const cdaLocation = ``;
-const converterBaseUrl = "http://localhost:8777";
-const fhirBaseUrl = "http://localhost:8888";
+const cdaLocation = `/Users/jonahkaye/Desktop/2024-01-23T08:02:29.892Z/test-patient-sample`;
+const converterBaseUrl = "http://localhost:8080";
+const fhirBaseUrl = "http://localhost:8889";
```

Comment:
these need to be reverted

Reply:
Will revert the converter baseURL, but the idea of changing the FHIR base URL is the linked fhir-server PR in the description. The script should point there instead of your main fhir server. 

Will revert the cda location and converter base url